src/js/planting.js

```diff
@@ -68,7 +68,7 @@ export default class extends EventEmitter {
       collection: this.session().objects(),
       app: this,
     });
-    this.toolbox = new Toolbox.View.Sidebar({
+    this.toolbox = new Sidebar({
```

Comment:
I think we shouldn't give up on toolbox name. Whole functionality of toolbox is closed in toolbox directory, but previously was splitted into few modules. Now its only sidebar and maybe we should rename it to `toolbox/toolbox.js` or just move into `module/toolbox.js`.